packages/shared/components/inputs/ClosableInput.svelte

```diff
@@ -1,47 +1,65 @@
 <script lang="typescript">
-    import { NumberInput, TextInput, ClosableInputContainer } from 'shared/components'
+    import { NumberInput, TextInput, Icon } from 'shared/components'
 
     export let value = ''
     export let open = false
     export let buttonElement: HTMLButtonElement
     export let label: string
     export let placeholder: string
+    export let error: string
     export let inputType: 'text' | 'number' = 'text'
 
     let inputElement: HTMLInputElement
     let hasFocus: boolean
 
-    function onClose() {
+    $: if (inputElement === null && buttonElement) {
+        buttonElement.focus()
+    }
+
+    function handleClose() {
```

Comment:
I'm missing a return type.

@maxwellmattryan  Should we add a coding convention to rename the button handlers like this to the following `onCloseClick()`?

```suggestion
    function handleClose(): void {
```